docs/variants/protectli_vp46xx/building-manual.md

```diff
@@ -50,24 +47,15 @@ VP4670. Versions v1.0.17 and older do not support VP4650 and VP4670 at all.
     cd 3rdparty/blobs/mainboard/
     git init
     git remote add origin <PROTECTLI_BLOBS_REPO>
-    git fetch origin && git checkout protectli_vault_cml_v1.0.19
+    git fetch origin && git checkout protectli_vault_cml_v1.1.0
     cd -
     ```
 
-1. Build the firmware v1.0.19 or newer:
-
-    + for VP4630 and VP4650
-
-        ```bash
-        ./build.sh vp4630_vp4650
-        ```
+1. Build the firmware v1.1.0 or newer:
 
-    + for VP4670
-
-        ```bash
-        ./build.sh vp4670
-        ```
+    ```bash
+    ./build.sh vp46xx
```

Comment:
1.1.0 is alredy out and uses old procedure. So it should be for firmware newer than v1.1.0

Reply:
Updated in a86d5c3aa62b89e7317fa821fac81c05af9626cb